torch/testing/_internal/distributed/rpc/jit/rpc_test.py

```diff
@@ -17,12 +17,33 @@
     RpcAgentTestFixture,
 )
 
+@torch.jit.script
```

Comment:
I tried similar code without RPC. And it works:

```python
import torch


@torch.jit.script
def get_dict():
    # type: () -> Dict[int, int]
    x: Dict[int, int] = {}
    x[1] = 2
    return x

print(get_dict())
```

Had a discussion with @xush6528, the error looks like the `TorchScript` function was compiled again when processing the RPC?